frontrunner_sdk/models/order.py

```diff
@@ -1,11 +1,45 @@
-from dataclasses import dataclass
-from typing import Literal
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Literal, Sequence
+
+from pyinjective.proto.exchange.injective_derivative_exchange_rpc_pb2 import DerivativeOrderHistory
 
 OrderType = Literal["buy", "sell", "stop_buy", "stop_sell", "take_buy", "take_sell", "buy_po", "sell_po"]
 OrderState = Literal["booked", "partial_filled", "filled", "canceled"]
 OrderExecutionType = Literal["limit", "market"]
 
 
+class FrOrderType(Enum):
+  BUY_LONG = 0
+  BUY_SHORT = 1
+  SELL_LONG = 2
+  SELL_SHORT = 3
+
+@dataclass
+class DerivativeOrderHistoryWrapper:
+  order: DerivativeOrderHistory
+  frOrderType: FrOrderType = field(init=False)
+
+  def __post_init__(self):
+    # https://docs.python.org/3.7/library/dataclasses.html#post-init-processing
+    self.frOrderType = self._order_to_fr_order_type(self.order)
```

Comment:
nit: snake case, since that's the style of other fields. in this case: `fr_order_type`.